airbyte-webapp/src/components/connectorBuilder/Builder/InputsView.tsx

```diff
@@ -95,14 +106,31 @@ export const InputsView: React.FC = () => {
       }),
     [inputInEditing?.isNew, inputInEditing?.key, usedKeys]
   );
+
   return (
     <BuilderConfigView heading={formatMessage({ id: "connectorBuilder.inputsTitle" })}>
       <Text centered className={styles.inputsDescription}>
         <FormattedMessage id="connectorBuilder.inputsDescription" />
       </Text>
-      {inputs.value.length > 0 && (
+      {(inputs.value.length > 0 || inferredInputs.length > 0) && (
         <Card withPadding className={styles.inputsCard}>
           <ol className={styles.list}>
+            {inferredInputs.map((input) => (
+              <li className={styles.listItem} key={input.key}>
+                <div className={styles.itemLabel}>{input.definition.title || input.key}</div>
+                <Button
+                  className={styles.itemButton}
+                  size="sm"
+                  variant="secondary"
+                  aria-label="Edit"
+                  onClick={() => {
+                    setInputInEditing(formInputToInputInEditing(input, true));
+                  }}
+                >
+                  <FontAwesomeIcon className={styles.icon} icon={faGear} />
+                </Button>
+              </li>
```

Comment:
This is almost the same code as the code below it (besides the `true/false` passed to formInputToInputInEditing) - can we extract it into its own component to reduce code duplication?

Reply:
Good call, feels cleaner anyway.